Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/CountryList.kt

```diff
@@ -2,47 +2,98 @@ package de.rki.coronawarnapp.ui.view
 
 import android.content.Context
 import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.LinearLayout
-import android.widget.TextView
+import androidx.core.content.withStyledAttributes
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import de.rki.coronawarnapp.R
 import de.rki.coronawarnapp.ui.Country
+import kotlinx.android.synthetic.main.view_country_list_entry_flag_container.view.*
 import java.text.Collator
 
 class CountryList(context: Context, attrs: AttributeSet) :
     LinearLayout(context, attrs) {
-
+    private var attributeSet = attrs
+    private var view: View? = null
     private var _list: List<Country>? = null
     var list: List<Country>?
```

Comment:
Can we make this non-nullable?

Reply:
the new changes didn't require the variable view. So should be resolved